.azure-pipelines/steps/install-dotnet.yml

```diff
@@ -21,4 +21,10 @@ steps:
   displayName: install dotnet core sdk 5.0
   inputs:
     packageType: sdk
-    version: $(dotnetCoreSdk5Version)
\ No newline at end of file
+    version: $(dotnetCoreSdk5Version)
```

Comment:
We only need the runtime version of .NET 5 here, as we have the .NET 6 SDK

Reply:
Done in https://github.com/DataDog/dd-trace-dotnet/pull/1885/commits/242c3c813e05991fdf1543aa6f7574ce577c6805